mri/operators/base.py

```diff
@@ -0,0 +1,40 @@
+# #############################################################################
+#  pySAP - Copyright (C) CEA, 2017 - 2018                                     #
+#  Distributed under the terms of the CeCILL-B license,                       #
+#  as published by the CEA-CNRS-INRIA. Refer to the LICENSE file or to        #
+#  http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.html for details.   #
+# #############################################################################
+
+
+class OperatorBase(object):
+    """ Base Operator class. Every operator inherits from this class,
```

Comment:
I don't know why you resolved (let me resolve the conversations I start for the next ones), but here I think we should specify that we are talking about linear operators (even if the notion of adjoint is larger than that).

Reply:
Ahh, Im sorry, I missed that this had you to answer.
Hmm makes sense..